seclang/parser.go

```diff
@@ -85,27 +85,27 @@ func (p *Parser) FromString(data string) error {
 		} else if inQuotes && len(line) > 0 && line[0] == '`' {
 			inQuotes = false
 		}
+		linebuffer += line
```

Comment:
While we're here some ideas

- `line := strings.TrimSpace` meaning it doesn't end in whitespace. Instead of using `pattern` to check for a trailing slash, I think it can just be `line[len(line)-1] == '\'`
- Instead of appending with \ and then removing, just append the slice
- Use `strings.Builder`

I think it ends up roughly like

```
linebuffer := strings.Builder{}

for {

  line := strings.TrimSpace(scanner.Text())
  if line == "" {
    continue
  }

  inQuotes = foo

  if inQuotes {
    linebuffer.WriteString(line)
    continue
  }

  if line[len(line)-1] == '\' {
    linebuffer.WriteString(line[0:len(line)-1])
    continue
  }

  linebuffer.WriteString(line)
  p.evaluateLine(linebuffer.String()
  linebuffer.Reset()
}